eu/intermodalics/tango_ros_common/maven-metadata.xml

```diff
@@ -3,13 +3,11 @@
   <groupId>eu.intermodalics</groupId>
   <artifactId>tango_ros_common</artifactId>
   <versioning>
-    <release>1.3.0</release>
+    <release>1.3.1</release>
     <versions>
-      <version>1.1.0</version>
```

Comment:
Why are the older versions being deleted from the index? I think they should still stay available, otherwise they won't be found. Was this done automatically by the tool? Are the older version artifacts still there? Could you please check to make sure things are still consistent?